### PR TITLE
[DELETE] /api/groups/{groupId}/students?ids={student ids s…

### PR DESCRIPTION
**Pull request by Wally5077**

### CODE REVIEW

**Review by Chaoyuuu — APPROVED** [review]
LGTM

File: Entities/src/main/java/tw/waterball/judgegirl/entities/Group.java [hunk 1]
```diff
@@ -53,11 +55,13 @@ public void deleteStudent(Student student) {
         student.getGroups().remove(this);
     }
 
-    public void deleteStudentById(int studentId) throws NotFoundException {
-        Student student = students.stream()
-                .filter(studentElement -> studentElement.id == studentId)
-                .findFirst()
-                .orElseThrow(NotFoundException::new);
-        deleteStudent(student);
+    public void deleteStudentByIds(String... ids) throws NotFoundException {
```
Comment (Johnny850807) [inline Entities/src/main/java/tw/waterball/judgegirl/entities/Group.java:58]: `public void deleteStudentByIds(int... ids)`

File: Student-Service/src/main/java/tw/waterball/judgegirl/studentservice/domain/usecases/group/DeleteStudentsFromGroupUseCase.java [hunk 2]
```diff
@@ -13,26 +13,26 @@
...
+        group.deleteStudentByIds(ids);
         groupRepository.save(group);
     }
 
     @NoArgsConstructor
     @AllArgsConstructor
     public static class Request {
         public int groupId;
-        public int studentId;
+        public String ids = "";
```
Comment (Johnny850807) [inline Student-Service/src/main/java/tw/waterball/judgegirl/studentservice/domain/usecases/group/DeleteStudentsFromGroupUseCase.java:35]: should be `int[] ids` or `List<Integer>`

File: Spring-Boot/Spring-Boot-Student/src/test/java/tw/waterball/judgegirl/springboot/student/controllers/it/GroupControllerTest.java [hunk 3]
```diff
@@ -320,4 +318,29 @@ private ResultActions addStudentsIntoGroupByMailList(int groupId, String[] mailL
                 .content(toJson(mailList)));
     }
 
+    @Test
+    public void GivenThreeStudentsAddedIntoCreatedGroup_WhenDeleteStudentsByIds_ShouldDeleteSuccessfully() throws Exception {
```
Comment (Johnny850807) [inline Spring-Boot/Spring-Boot-Student/src/test/java/tw/waterball/judgegirl/springboot/student/controllers/it/GroupControllerTest.java:322]: `GivenStudents_A_B_C_AddedIntoGroup_WhenDeleteStudentsByIds_A_B_ShouldRemainStudentCInGroup`

**Review by Johnny850807 — CHANGES_REQUESTED** [review]
Review done!

**Review by Johnny850807 — APPROVED** [review]
LGTM

**Pull request merged**